public/less/themes/dark.less

```diff
@@ -0,0 +1,752 @@
+@import "_base";
```

Comment:
Please rename this to arc-green as this is what the theme from #730 was based on.

Reply:
updated